package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/keep-network/run-workflow#readme",
   "dependencies": {
     "@actions/core": "^1.2.6",
-    "@keep-network/ci": "github:keep-network/ci"
+    "@keep-network/ci": "https://github.com/keep-network/ci#rfc-18/integration"
```

Comment:
This should remain unchanged, right?

Reply:
Right, see cee5782d066fb54cbe0f77f4a2a1f5db57e3468b.